workspace/solution.go

```diff
@@ -87,3 +83,37 @@ func (s *Solution) PathToParent() string {
 	}
 	return filepath.Join(dir, s.Track)
 }
+
+// SolutionMetadataFilepath is the path of the solution metadata file relative to the workspace.
+func SolutionMetadataFilepath() string {
```

Comment:
I don't understand how this works. Each exercise has its own metadata file, so this is relative to a subdirectory within the workspace.

Reply:
I mistakenly used 'workspace' when I meant 'exercise' there.

Reply:
Ok, that makes sense. Thanks!